charts/argo-cd/crds/crd-project.yaml

```diff
@@ -164,7 +164,7 @@ spec:
                       description: Name is a name for this role
                       type: string
                     policies:
-                      description: Policies Stores a list of casbin formated strings that define access policies for the role in the project
```

Comment:
I don't think we should edit this one here, since it's pulled in directly from the upstream CRDs

Reply:
I agree, CRDs should be fixed upstream.

Reply:
Sure, got a handy link? otherwise i'll figure it out.

Reply:
I guess the files to modify are here (I don't think they are otherwise generated): https://github.com/argoproj/argo-cd/tree/master/manifests/crds

Reply:
https://github.com/argoproj/argo-cd/pull/5373